app/models/manageiq/providers/oracle_cloud/cloud_manager/event_catcher/runner.rb

```diff
@@ -40,11 +39,11 @@ def filtered?(event)
   end
 
   def ensure_event_stream
```

Comment:
Maybe not for this PR, but this can just become `def event_stream` directly...no need for the attr_reader.

Reply:
Yeah I considered that but since we're calling this before the main loop to set up the stream I thought it'd look weird to just call `event_stream`.  That might not be necessary in this case and we could just drop that and leave it as `event_stream.poll ...`